### ISSUE TITLE
Git module: get_remote_head() takes exactly 6 arguments (5 given)

### ISSUE THREAD

**Issue by vwall** [ansible/ansible#5036]
Version: ansible 1.4

When using the git module and no gitconfig file 

File: 
https://github.com/ansible/ansible/blob/ad5012f930a50d64209e662945a3fe20e8ae02b9/library/source_control/git#L385

```
remote_head = get_remote_head(git_path, module, dest, version, repo)
```

only supplies 5 arguments when get_remote_head requires 6.

**Comment by mpdehaan** [comment]
Can you please provide the playbook line you used to reproduce this problem?

Thanks!

**Comment by vwall** [comment]
```
- name: Git Repos
  git: repo={{ item.repo }} dest={{ item.dest }}
  with_items: dotfiles.vwall.git.repos
```

**Comment by jctanner** [comment]
Fixed by d950a3d1b5e46705b96d5d2af78b0b3f3f37c24f
